### PR TITLE
Updated client configurations

### PR DESCRIPTION
**Pull request by AkhileshSreelayam**
Main changes are:

- Updated self sign certificate process in all sample configurations (trusting Let’s Encrypt trustchain)
- Updated .NET target version to 8.0
- Updated READMEs

### CODE REVIEW

**Review by JoernSchimmelpfeng — CHANGES_REQUESTED** [review]
Please add your entry of the maintainers document.

**Comment by AkhileshSreelayam** [comment]
> Please add your entry of the maintainers document.

made the change.

**Review by davidpiendl — APPROVED** [review]

File: kafka-sample/c#/README.md [hunk 1]
```diff
@@ -17,46 +18,33 @@ Package Links:
...
-mkdir -p /etc/pki/tls/certs
-ln -s /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
+sudo update-ca-certificates
 ```
 
 How to use
 ----------
 
-In order to use the sample please change the following parameters. Those parameters should previously send to you.
+In order to use the sample, please substitute the following parameters with those you have received via secure email.
```
Comment (JoernSchimmelpfeng) [inline kafka-sample/c#/README.md:34]: I propose: 

To use the sample, please change the following parameters, which should have been sent to you beforehand.

File: kafka-sample/c#/README.md [hunk 2]
```diff
@@ -17,46 +18,33 @@ Package Links:
...
-You also need to provide the CA certificate location.
-
-```cs
-  var rootCaFile = @"cluster-ca.crt";                 // file path of your CA certificate (must be a PEM file)
+  var clientName = "YOUR_CLIENT_NAME";                  // use the client name you have received 
+  var clientId = "YOUR_CLIENT_ID";                      // use the client id you have received
+  var clientSecret = "YOUR_CLIENT_SECRET";              // use the secret you have received
+  var scope = "SCOPE";                                  // use the correct scope for your region
+  var topic = $"vehiclesignals.{clientName}";           // use topic for the client you have received
+  var consumerGroup = $"{clientName}.GROUP_ID_POSTFIX"; // you can change the postfix of your consumer group
```
Comment (JoernSchimmelpfeng) [inline kafka-sample/c#/README.md:42]: I guess this is MBCon specific. We should not treat it as a general approach.

File: kafka-sample/golang/README.md [hunk 3]
```diff
@@ -22,46 +22,37 @@ go install
...
-mkdir -p /etc/pki/tls/certs
-ln -s /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
+sudo update-ca-certificates
 ```
 
 How to use
 ----------
 
-In order to use the sample please change the following parameters.
+In order to use the sample, please substitute the following parameters with those you have received via secure email.
```
Comment (JoernSchimmelpfeng) [inline kafka-sample/golang/README.md:38]: see above.

File: kafka-sample/java/README.md [hunk 4]
```diff
@@ -23,26 +23,28 @@ you can use the gradle `shadowJar` task to build an executable [jarfile](build/l
 gradle shadowJar
 ```
 
+
+Our Kafka server uses Let’s Encrypt CA for SSL/TLS certificates. These certificates are usually included in the default
```
Comment (JoernSchimmelpfeng) [inline kafka-sample/java/README.md:27]: How is our? Please use something like "In this example we assume to use Lets Encrypt..."

**Review by JoernSchimmelpfeng — CHANGES_REQUESTED** [review]
Please make sure the example can also be used for other use cases beside MBCon.

**Review by JoernSchimmelpfeng — APPROVED** [review]

**Comment by JoernSchimmelpfeng** [comment]
ATTENTION: This PR may be merged. But should only be done when the IDP switch has been completed.

File: kafka-sample/java/README.md (hunk 4 above)
Comment (AkhileshSreelayam) [reply kafka-sample/java/README.md]: implemented as suggested.

File: kafka-sample/golang/README.md (hunk 3 above)
Comment (AkhileshSreelayam) [reply kafka-sample/golang/README.md]: Implemented as suggested.

File: kafka-sample/c#/README.md (hunk 2 above)
Comment (AkhileshSreelayam) [reply kafka-sample/c#/README.md]: Implemented as suggested.

File: kafka-sample/c#/README.md (hunk 1 above)
Comment (AkhileshSreelayam) [reply kafka-sample/c#/README.md]: Implemented as suggested.

**Pull request merged**